client/src/components/Billing/index.js

```diff
@@ -159,13 +164,11 @@ function Pricing(props) {
                   ))}
                 </CardContent>
                 <CardActions className={classes.cardActions}>
-                  <Button
-                    fullWidth
-                    variant={tier.buttonVariant}
-                    color="primary"
-                  >
-                    {tier.buttonText}
-                  </Button>
+                  
+                  <Checkout
```

Comment:
Button needs to be aligned center if you can
And maybe a bit longer I would say